configs/gateway-server/gateway-server.yml

```
@@ -0,0 +1,69 @@
+          uri: lb://notification-service
+          predicates:
+            - Path=/api/v1/slackmessages/**
+
+        - id: product-service
+          uri: lb://product-service
+          predicates:
+            - Path=/api/v1/products/**
+
+# Keycloak 연동 설정
+# 이 설정은 Gateway에서 직접 JWT를 검증할 때 사용됩니다.
+spring:
```

YAMLlint (1.38.0)

[error] duplication of key "spring" in mapping

(key-duplicates)